utils/neuralmagic/utils.py

```diff
@@ -113,6 +114,78 @@ def nm_log_console(message: str, logger: "Logger" = None, level: str = "info"):
             logger.info(f"{colorstr('Neural Magic: ')}{message}")
 
 
+def neuralmagic_onnx_export(
+    model: torch.nn.Module,
+    sample_data: torch.Tensor,
+    weights_path: Path,
+    one_shot: Optional[str],
+    dynamic: Optional[Dict[str, int]],
+    output_names: List[str],
+) -> Path:
+    """
+    Augmented ONNX export to optimize and properly post-process sparsified models
+
+    :param model: model to export
+    :param sample_data: data to be used with export
+    :weights path: path from which the torch model was loaded. Used only for save
+        pathing and naming purposes
+    :one_shot: one_shot recipe, if one was applied. Used only for save pathing and
+        naming purposes
+    :param dynamic: dictionary of input or output names to list of dimensions
+        of those tensors that should be exported as dynamic
+    :output_names: names of output tensors
+
```

Comment:
nit: maybe not so relevant for this repo, but anyway:
```suggestion
```

Reply:
Haha good catch